.api/v1.5.0/schema.pbiviz.json

```diff
@@ -0,0 +1,95 @@
+{
```

Comment:
I think that a more proper way is to use local version of PBIVIZ instead of global one. It means that we should include PBIVIZ to [dev-dependencies](https://github.com/Microsoft/PowerBI-visuals-ForceGraph/blob/master/package.json#L49)

As a result developer's workflow is going to be changed:
* git clone repo
* npm install
* npm start // to start a debug mode. This command uses the local PBIVIZ